Javascript/Age-ify (A future age calculator)/age.js

```diff
@@ -0,0 +1,12 @@
+let yearOfBirth = 1999;
+let yearFuture = 2049;
+
+let age = yearFuture - yearOfBirth;
```

Comment:
You can set those variables with a `const`. Use `let` only when you are to reassign their values.

Reply:
***Got it. Changed.***